projects/standard-rulesets/src/breaking-changes/preventArrayItemLengthChanges.ts

```diff
@@ -0,0 +1,77 @@
+import {
+  RequestRule,
+  ResponseBodyRule,
+  RuleError,
+} from '@useoptic/rulesets-base';
+import {
+  arrayMaxItemsShouldNotIncrease,
+  arrayMinItemsShouldNotDecrease,
+  numericMaximumRangeShouldNotExpand,
+  numericMinimumRangeShouldNotExpand,
+} from './helpers/rangeChange';
+
+export const preventRequestArrayLengthChanges = new RequestRule({
+  name: 'prevent request array length changes',
+  rule: (requestAssertions) => {
+    requestAssertions.body.changed((before, after) => {
```

Comment:
@niclim is this correct? I want both of these to run independently of each other, ie if Max throws, min still runs.

Reply:
yes both will run